app/services/oneoffs/flipflop_users_cleanup.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Oneoffs
+  class FlipflopUsersCleanup
+    def call
+      result = []
+
+      flip_flop_users.each do |from_user|
+        if from_user.archived?
+          result << [from_user.id, "User is archived"]
```

Comment:
I don't feel all that strongly about it, but I started trying to standardise how we record information in these one off scripts - if you want to use it there's a concern here:

https://github.com/DFE-Digital/early-careers-framework/blob/main/lib/has_recordable_information.rb